documentation/business_care_intitule.md

```
@@ -0,0 +1,370 @@
+<tr><td colspan="5">Conseils hebdomadaires </td><td colspan="1">non </td><td colspan="3">oui (non personnalisés) </td><td colspan="1">oui personnalisés (suggestion d'activités) </td></tr>
+<tr><td colspan="5">Evénements / Communautés (sans intervention des prestataires de BC, événements internes de l'entreprise) </td><td colspan="5">accès illimité </td></tr>
+</table>
+
+
+
+|Tarif annuel par salarié |180 € |150 € |100 € |
+| - | - | - | - |
+
+***ANNEXE 2 SCHEMA RESEAU INITIAL*** 
+
+![](Aspose.Words.29680396-ac9d-4cad-8357-49b370a89546.002.jpeg)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)